include/roboteam_ai/stp/Play.hpp

```diff
@@ -103,11 +114,17 @@ class Play {
      */
     rtt::ai::Field field;
 
-    protected:
     /**
      * Assigns robots to roles
      */
     virtual void assignRoles() noexcept = 0;
+
+    /**
+     * This function is used to determine if, when a role is in an endtactic, that endtactic should be skipped.
+     * An example could be BlockRobot and Intercept. You block a robot until a ball is shot and then the robot
+     * closest to the ball should try to intercept
+     */
+    virtual bool shouldRoleSkipEndTactic() = 0;
```

Comment:
I feel this needs more information. The world? The current robot? The role that you apply this to? Something

Reply:
Probably, but if it does you can just add it right?